docs/hub/webhooks.md

```diff
@@ -0,0 +1,193 @@
+# Webhooks
+
+Webhooks are a foundation for MLOps related features. You can use this to auto-convert models, build community bots, or build CI/CD for your datasets or Spaces…
+
+They allow you to react when new changes happen on specific repos or repos belonging to specific users / organizations (not just your repos, but any repo!).
+
+You can configure the webhooks in your [settings](https://huggingface.co/settings/webhooks):
+
+![Settings of an individual webhook](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/webhook-settings.png)
+
+You can watch repo changes and discussions / pull requests / comments. You can even use a Space to react to webhooks!
+
+<!-- Todo: add a link to a guide with a real example -->
+
+## Webhook Payloads
+
+After registering a webhook, you will be notified of events via an `HTTP POST` call on the specified URL. The payload is encoded in JSON.
+
+You can check the last payloads sent in the activity tab of the webhook page, as well as replay webhooks:
+
+
+![image.png](https://s3.amazonaws.com/moonup/production/uploads/1671034300077-61d2f90c3c2083e1c08af22d.png)
+
+As an example, here is the full payload when a pull request is opened:
+
+```json
+{
+  "event": {
+    "action": "create",
+    "scope": "discussion"
+  },
+  "repo": {
+    "type": "model",
+    "name": "gpt2",
+    "id": "621ffdc036468d709f17434d",
+    "private": false,
+    "url": {
+      "web": "https://hugginface.co/gpt2",
+      "api": "https://hugginface.co/api/models/gpt2"
+    },
+    "owner": {
+      "id": "628b753283ef59b5be89e937"
+    }
+  },
+  "discussion": {
+    "id": "6399f58518721fdd27fc9ca9",
+    "title": "Update co2 emissions",
+    "url": {
+      "web": "https://hugginface.co/gpt2/discussions/19",
+      "api": "https://hugginface.co/api/models/gpt2/discussions/19"
+    },
+    "status": "open",
+    "author": {
+      "id": "61d2f90c3c2083e1c08af22d"
+    },
+    "num": 19,
+    "isPullRequest": true,
+    "changes": {
+      "base": "refs/heads/main"
+    }
+  },
+  "comment": {
+    "id": "6399f58518721fdd27fc9caa",
+    "author": {
+      "id": "61d2f90c3c2083e1c08af22d"
+    },
+    "content": "Add co2 emissions information to the model card",
+    "hidden": false,
+    "url": {
+      "web": "https://hugginface.co/gpt2/discussions/19#6399f58518721fdd27fc9caa"
+    }
+  },
+  "webhook": {
+    "id": "6390e855e30d9209411de93b",
+    "version": 3
+  }
+}
+```
+
+### Event
+
+The top-level properties `event` is always specified and used to determine the nature of the event.
+
+It has two sub-properties: `event.action` and `event.scope`.
+
+`event.scope` can take one of the following values:
+
+- `"repo"` - Global events on repos. Possible values for the associated `action`: `"create"`, `"delete"`, `"update"`, `"move"`.
```

Comment:
I wonder if a better format for this would be a table with three columns: event.scope value, description, associated actions